benches/sst2_benchmark.rs

```diff
@@ -83,8 +83,9 @@ fn bench_sst2(c: &mut Criterion) {
         torch_sys::dummy_cuda_dependency();
     }
     //    Define input
-    let mut sst2_path = PathBuf::from(env::var("SST2_PATH")
-        .expect("Please set the \"squad_dataset\" environment variable pointing to the SQuAD dataset folder"));
+    let mut sst2_path = PathBuf::from(env::var("SST2_PATH").expect(
+        "Please set the \"SST2_PATH\" environment variable pointing to the SST2 dataset folder",
```

Comment:
I just fixed this because I saw the typo while running the benchmarks but I really think these paths should point directly to files instead of directories. Happy to make a PR for that if you agree!